subjects/DPxAI/first-function/README.md

```diff
@@ -0,0 +1,87 @@
+## First Function
+
+> Mindful AI mode
+
+### Context
+
+Your robot will need to start executing custom tasks, and to do that, you'll need to program its `functions`. Functions are like commands that tell robot exactly what to do.
+
+You can create your own functions to give your robot unique abilities and make it more efficient!
+
+### AI-Powered Learning Techniques
+
+`Reflective Practice Technique:`
+This type of prompt encourages learners to reflect on the concepts they’ve just learned, reinforcing their understanding by applying the concepts in different contexts or scenarios.
```

Comment:
I find it odd that the subject is not treating the reader by "you" as it did above. It seems like this sentence was made to explain the exercise to the campus admin instead of the learner

Reply:
It is changed now.